datafusion/core/src/physical_plan/sorts/mod.rs

```diff
@@ -17,30 +17,14 @@
 
 //! Sort functionalities
 
-use crate::physical_plan::SendableRecordBatchStream;
-use std::fmt::{Debug, Formatter};
-
+mod builder;
 mod cursor;
 mod index;
+mod merge;
 pub mod sort;
 pub mod sort_preserving_merge;
+mod stream;
 
 pub use cursor::SortKeyCursor;
 pub use index::RowIndex;
-
-pub(crate) struct SortedStream {
```

Comment:
This abstraction was a tad pointless, it was only used by `ExternalSorter` to propagate the size of the in-memory sorted batches. I adjusted it to just call init_mem_used itself, and created #5885 to track the broader pre-existing issue of memory accounting within merge streams